test/index.html

```diff
@@ -18,7 +18,10 @@
 <body>
   <script>
     WCT.loadSuites([
-      'lazy-imports_test.html',
+      '1.x/lazy-imports_test.html?dom=shadow',
```

Comment:
- 1.x/lazy-imports_test.html
 - 1.x/lazy-imports_test.html?dom=shadow
- 2.0/lazy-imports_test.html?wc-shadydom=true&wc-ce=true
- 2.0/lazy-imports_test.html

In our hybrid elements we generalize this as:
- lazy-imports_test.html?wc-shadydom=true&wc-ce=true
- lazy-imports_test.html?dom=shadow